arwendebug/facade_test.go

```diff
@@ -0,0 +1,114 @@
+package arwendebug
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var databasePath = "./testdata/db"
+var wasmCounterPath = "../test/contracts/counter/counter.wasm"
+var wasmErc20Path = "../test/contracts/erc20/erc20.wasm"
+
+func init() {
+	os.RemoveAll(databasePath)
```

Comment:
Unhandled error, if we don't want to treat it maybe just put `_ = ...` so we are explicit about it

Reply:
Fixed.